src/flp.rs

```diff
@@ -111,11 +111,6 @@ pub enum FlpError {
     /// An error happened during noising.
     #[error("differential privacy error: {0}")]
     DifferentialPrivacy(#[from] crate::dp::DpError),
-
-    /// Unit test error.
-    #[cfg(test)]
-    #[error("test failed: {0}")]
-    Test(String),
```

Comment:
Question: The new tool replaces errors with panics. Is this useful? If you think it would be better to return errors, let me know and I'll revert.

Reply:
This sounds great, immediately panicking is more appropriate for tests than returning an error (backtraces will be more informative). Plus, I'm always happy to remove conditional compilation from an enum's definition.